docs/transform/modernize-userinterface-site-pages-api.md

```diff
@@ -523,9 +523,9 @@ Name|Description
 
 #### Input parameters
 
-Name|Description
-:-----|:----------
-{ContactLoginName}|User account to lookup (in i:0#.f|membership|joe@contoso.onmicrosoft.com format)
+Name|Description|
+:-----|:----------|
+|{ContactLoginName}|User account to lookup (in i:0#.f\|membership\|joe@contoso.onmicrosoft.com)
```

Comment:
What happened to the word "format"? The way this has been left doesn't make sense